### ISSUE TITLE
Please fix non-normative example for refresh_token_expires_at and sharing_expires_at

### ISSUE THREAD

**Issue by spikejump** [ConsumerDataStandardsAustralia/standards-maintenance#180]
# Description
Both refresh_token_expires_at and sharing_expires_at are defined as NumericDate values.  However, the non-normative example for them shows the values being as strings.

```
{
  "iss": "https://www.holder.com.au",
  "sub": "a9ebbef6-1f0b-44eb-96cf-0c5b51b37ab2",
  "aud": "12345",
  "nonce": "n-0S6_WzA2Mj",
  "exp": 1311281970,
  "iat": 1311280970,
  "nbf": 1311280970,
  "auth_time": 1311280969,
  "acr": "urn:cds.au:cdr:3",
  "refresh_token_expires_at": "1311281970",
  "sharing_expires_at": "1311281970"
}
```

# Change Proposed
Please update the non-normative example to be numeric date values.  For example,
```
{
  "iss": "https://www.holder.com.au",
  "sub": "a9ebbef6-1f0b-44eb-96cf-0c5b51b37ab2",
  "aud": "12345",
  "nonce": "n-0S6_WzA2Mj",
  "exp": 1311281970,
  "iat": 1311280970,
  "nbf": 1311280970,
  "auth_time": 1311280969,
  "acr": "urn:cds.au:cdr:3",
  "refresh_token_expires_at": 1311281970,
  "sharing_expires_at": 1311281970
}
```

**Comment by perlboy** [comment]
+1 on this, among other reasons this is important because Nimbus OAuth2 client handles `getCustomParameters` on token responses as `Map<String,Object>` which means class casting is explicitly defined on payload type resulting in class casting exceptions for non compliant holders.

**Comment by CDR-API-Stream** [comment]
This documentation defect was fixed in v1.3.1 of the data standards. 

This issue will be closed accordingly.
